lib/virtfs/xfs/file.rb

```diff
@@ -0,0 +1,14 @@
+module VirtFS::XFS
+  class File
```

Comment:
Just a note - should I assume you've made this same comment, perhaps offline, to @movitto regarding https://github.com/ManageIQ/virtfs-ext3/pull/1 ? This code was modified from that PR.